shared/src/commonMain/kotlin/io/customer/shared/tracking/api/model/BatchTracking.kt

```diff
@@ -0,0 +1,36 @@
+package io.customer.shared.tracking.api.model
+
+import io.customer.shared.tracking.model.TrackingError
+import kotlinx.serialization.SerialName
+
+/**
+ * Request class only to be used when making tracking requests in batch.
+ */
+@kotlinx.serialization.Serializable
+internal data class BatchTrackingRequestBody(
+    @SerialName("batch") val batch: List<TrackingRequest>,
+)
+
+/**
+ * Response class only to be used when making tracking requests in batch.
+ */
+@kotlinx.serialization.Serializable
+internal data class BatchTrackingResponseBody(
+    @SerialName("meta") val meta: BatchTrackingResponseMeta?,
+    @SerialName("errors") val errors: List<TrackingError>?,
```

Comment:
```suggestion
    val meta: BatchTrackingResponseMeta?,
    val errors: List<TrackingError>?,
```

Reply:
Responded above, won't be addressing it in this PR.